client/app/locales/en.json

```diff
@@ -98,6 +98,23 @@
   "account identifiers": "Account",
   "account personalization": "Customization",
   "account password": "Password",
+  "account two factors auth": "Two-factors authentication",
```

Comment:
The right name is "Tow-factor authentication".

https://en.wikipedia.org/wiki/Two-factor_authentication